definitions/npm/file-type_v3.x.x/flow_>=v0.28.x/file-type_v3.x.x.js

```diff
@@ -0,0 +1,3 @@
+declare module 'file-type' {
+  declare module.exports: (buffer: Buffer | Uint8Array) => ?{ ext: string, mime: string };
```

Comment:
Well technically there's a difference because `?` says it can be `null` or `undefined`.

That doesn't have a real impact so just ignore me